src/styles/components/main.scss

```diff
@@ -0,0 +1,19 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
Why are these styles separate from card.scss file?

Reply:
because it doesn't work right (Idk why)

I tried(